src/iarc7_motion/iarc_tasks/task_utilities/translate_stop_planner.py

```diff
@@ -106,14 +111,8 @@ def _calculate_triangle_acceleration(self,
             # otherwise just keep accelerating
             if required_deceleration < self._desired_acceleration:
                 # If these two things are true we've been decelerating too fast
-                if (self._acceleration_went_negative and
```

Comment:
Can someone explain why this was removed and why it isn’t needed anymore?

Reply:
I don't remember why I removed it

Reply:
@amiller27 @aftersomemath do either of you know why?

Reply:
I said to remove it because it doesn't really make sense for a changing endpoint.  This is fine for now; we might need some extra stuff later if we have problems with oscillations near the endpoint, but that's out of scope.